src/scss/_mixins.scss

```diff
@@ -9,3 +9,13 @@
 		color: oColorsGetColorFor(link, text);
 	}
 }
+
+/// Target both IE8 and modern browsers at the M layout
+@mixin _oTechdocsHeaderBreakpoint {
```

Comment:
This is duplicated in https://github.com/Financial-Times/o-techdocs/pull/56/files#diff-9d9af257fc3565cf0d18f502422f236fR46 does it need to be?